src/main/java/no/nav/foreldrepenger/regler/uttak/fastsetteperiode/betingelser/SjekkOmErAlleDisponibleDagerIgjenMinsterett.java

```diff
@@ -0,0 +1,37 @@
+package no.nav.foreldrepenger.regler.uttak.fastsetteperiode.betingelser;
+
+
+import static no.nav.foreldrepenger.regler.uttak.fastsetteperiode.betingelser.SjekkOmTomForAlleSineKontoer.hentSøkerSineKontoer;
+
+import no.nav.foreldrepenger.regler.uttak.fastsetteperiode.FastsettePeriodeGrunnlag;
+import no.nav.foreldrepenger.regler.uttak.fastsetteperiode.grunnlag.Stønadskontotype;
+import no.nav.fpsak.nare.doc.RuleDocumentation;
+import no.nav.fpsak.nare.evaluation.Evaluation;
+import no.nav.fpsak.nare.specification.LeafSpecification;
+
+@RuleDocumentation(SjekkOmErAlleDisponibleDagerIgjenMinsterett.ID)
+public class SjekkOmErAlleDisponibleDagerIgjenMinsterett extends LeafSpecification<FastsettePeriodeGrunnlag> {
+
+    public static final String ID = "FP_VK 10.5.4.1";
+    public static final String BESKRIVELSE = "Er alle disponible dager minsterett?";
+
+    public SjekkOmErAlleDisponibleDagerIgjenMinsterett() {
+        super(ID);
+    }
+
+    @Override
```

Comment:
Trenger man disse sjekkene ? Minsterettlogikken i 10.5.4 er ikke knyttet mot stønadskonto.
Konteksten er tompåallekonto + bfhr. Burde det ikke da holde med 10.5.4
Ser for meg ut som en variant av nettoSaldoJustertForMinsterett med siste param false (tompåallekonto)

Reply:
Et par ting:

- FP_VK 10.5.4 sjekker om perioden om den aktuelle perioden "gjelder minsteretten". Siden dette er msp så blir resultatet alltid "nei"
- restSaldoMinsterett returnerer positiv saldo selv om alle dagene med foreldrepenger er 0
Mulig 10.5.4 heller kan endres?